proto/babylon/btcstaking/v1/incentive.proto

```diff
@@ -9,9 +9,15 @@ option go_package = "github.com/babylonlabs-io/babylon/x/btcstaking/types";
 // VotingPowerDistCache is the cache for voting power distribution of finality providers
 // and their BTC delegations at a height
 message VotingPowerDistCache {
+    option (gogoproto.goproto_getters) = false;
+    // total_voting_power is the total voting power of all the finality providers
+    // in the cache
     uint64 total_voting_power = 1;
     // finality_providers is a list of finality providers' voting power information
     repeated FinalityProviderDistInfo finality_providers = 2;
+    // num_active_fps is the number of finality providers that have positive voting power
```

Comment:
```suggestion
    // num_active_fps is the number of finality providers that have active BTC delegations
```

a FP has positive voting power at a height only when it has active BTC
delegations at this height AND has timestamped public randomness at this height

Reply:
Agreed. The two conditions combined lead to fp having voting power